src/web.rs

```diff
@@ -268,7 +268,7 @@ pub fn send_assets(
 #[wasm_bindgen]
 pub fn accept_transfer(consignment: String, blinding_factor: String, outpoint: String) -> Promise {
     set_panic_hook();
-
+    crate::info!("aqui estamos");
```

Comment:
Maybe set a more informative log message if you want to leave this in.

Reply:
No, i left several debug logs that are unnecessary...